sdk/iot/hub/samples/README.md

```diff
@@ -23,7 +23,32 @@ This document describes how to use samples and what is done in each sample.
 
 ## Getting started
 
-Getting started explained in detail [here][SDK_README_GETTING_STARTED].
+These samples use X509 authentication to connect to IoT Hub. To easily run these samples, we have provided
+two scripts: one to generate the root cert and device cert used to connect, and the other to generate the
+validation cert which IoT Hub uses to certify you are the owner of the root cert. The following details how
+to use these scripts.
+
+### `generate_certificate.sh`
+1. Create a device in IoT Hub with `X.509 CA Signed` authentication.
+1. Run the script using the following form:
+```bash
+./generate_certificate <device_id>
+```
+where you substitute you own device_id for `<device_id>`.
+
+After this is generated, the environment variable `AZ_IOT_DEVICE_X509_CERT_PEM_FILE` will be set for you
+and is ready to use in the samples.
+
+### `generate_validation.sh`
+IoT Hub has a validation step to ensure you own the CA cert. Once you acquire the validation code in the IoT
```

Comment:
When you mention "Once you acquire the validation code in the IoT Hub portal", I do believe there is a jump here that customers might have some trouble to follow. 
Maybe this link could be added, in case we do not want to repeat the instructions: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-x509ca-overview#how-to-register-the-x509-ca-certificate-to-iot-hub

Reply:
added